### ISSUE TITLE
No way to override primeng icons

### ISSUE THREAD

**Issue by pantonis** [primefaces/primeng#6007]
If you have a PrimeNG PRO Support subscription please post your issue at;

https://pro.primefaces.org 

where our team will respond within 4 business hours.

If you do not have a PrimeNG PRO Support subscription, fill-in the report below. Please note that
your issue will be added to the waiting list of community issues and will be reviewed on a first-come first-serve basis, as a result, the support team is unable to guarantee a specific schedule on when it will be reviewed. Thank you for your understanding.

Current Waiting Time: ~8 weeks.

**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

There is no way to use custom icons (font-awesome) in place of primeng icons. One example is the sidebar component. I cannot set custom close button icon. Same with turbotable sort icon. No way to override this

**Comment by glenr8** [comment]
I've replaced most of the icons with Material Design Icon files in one of my projects. I simply reference the Material Design css CDN in my index.html and changed the following .fa, .pi { font-family: "Material-Design-Iconic-Font", "primeicons"; } and it works for me.

Took quite a bit of time, and I only did this as we use material design fonts for all the other icons in our project and made sense to only use one. But in saying that I still needed primeicons and font-awesome as there were a few gaps where material design didn't have a suitable replacement.

Before the latest release PrimeNG used font-awesome, and it looks like a lot of the icons still require it even though their reference website doesn't say to include it.
